comid/rawvalue.go

```diff
@@ -41,6 +44,36 @@ func (o RawValue) GetBytes() ([]byte, error) {
 	}
 }
 
+func (o RawValue) Equal(r RawValue) bool {
+	return reflect.DeepEqual(o, r)
+}
+
```

Comment:
Could you please add a header comment to this method and reference the comparison section!

Reply:
OK, will do